data/kernels/negadoctor.cl

```diff
@@ -0,0 +1,53 @@
+/*
+    This file is part of darktable,
```

Comment:
As for C, header should be:

This file is part of darktable,
Copyright (C) 2020 darktable developers.

Reply:
You missed this one.